core/pkg/ingress/types.go

```diff
@@ -134,6 +135,8 @@ type Backend struct {
 	Secure bool `json:"secure"`
 	// Endpoints contains the list of endpoints currently running
 	Endpoints []Endpoint `json:"endpoints"`
+	// StickySession contains the StickyConfig object with stickness configuration
+	StickySession stickysession.StickyConfig `json:"stickysession,omitempty"`
```

Comment:
I agree with you. I forgot that Haproxy can make this through IP instead of cookies (like when you're load balancing TCP or UDP only).